src/index.js

```diff
@@ -0,0 +1,96 @@
+const state = {
+    tempVar : 76,
+    skyDisplay : {
+        'Sunny' : '☀️☀️☀️☀️☀️☀️☀️☀️☀️☀️',
+        'Cloudy' : '☁️☁️☁️☁️☁️☁️☁️☁️☁️☁️',
+        'Rainy' : '🌧🌧🌧🌧🌧🌧🌧🌧🌧🌧',
+        'Stormy' : '⛈⛈⛈⛈⛈⛈⛈⛈⛈⛈',
+        'Snowy' : '🌨🌨🌨🌨🌨🌨🌨🌨🌨🌨',
+        'Gay' : '🌈🌈🌈🌈🌈🌈🌈🌈🌈🌈',
+        'Meatball' : '🍝🍝🍝🍝🍝🍝🍝🍝🍝🍝'
+    },
+    landscapeDisplay : {
+        'Hot' : '🌵🌵🌵🌵🌵🌵🌵🌵🌵🌵',
+        'Warm' : '🌴🌴🌴🌴🌴🌴🌴🌴🌴🌴',
+        'Meh' : '🍂🍂🍂🍂🍂🍂🍂🍂🍂🍂',
+        'Cold' : '❄️❄️❄️❄️❄️❄️❄️❄️❄️❄️',
+        'Freezing' : '☃️☃️☃️☃️☃️☃️☃️☃️☃️☃️'
+    },
+    cityName : ''
+};
+
+const upTemp = (event) => {
+    state.tempVar += 1;
+    const tempDisplay = document.getElementById("temp-display-big");
+    tempDisplay.textContent = `${state.tempVar} F`;
+    tempChecker(state.tempVar, tempDisplay);
```

Comment:
These lines are identical to the final 3 lines in `downTemp`. Consider moving them to a helper and calling it from both.